roles/chrony/defaults/main.yml

```diff
@@ -6,3 +6,8 @@ public_ntp_servers:
   - time2.google.com
   - time3.google.com
   - time4.google.com
+
+chrony_package: default
```

Comment:
I'd use DEFAULT (uppercase) to make it clear at a first glance that this is a special value.